src/main/java/org/aksw/iguana/cc/tasks/impl/Stresstest.java

```diff
@@ -0,0 +1,111 @@
+package org.aksw.iguana.cc.tasks.impl;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.aksw.iguana.cc.metrics.Metric;
+import org.aksw.iguana.cc.storage.Storage;
+import org.aksw.iguana.cc.tasks.Task;
+import org.aksw.iguana.cc.worker.HttpWorker;
+import org.aksw.iguana.cc.worker.ResponseBodyProcessorInstances;
+import org.aksw.iguana.cc.worker.impl.SPARQLProtocolWorker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.concurrent.*;
+
+
+/**
+ * Stresstest.
+ * Will stresstest a connection using several Workers (simulated Users) each in one thread.
+ */
+public class Stresstest implements Task {
+
+    public record Config(
+            List<HttpWorker.Config> warmupWorkers,
+            @JsonProperty(required = true) List<HttpWorker.Config> workers
+    ) implements Task.Config {}
+
+    public record Result(
+            List<HttpWorker.Result> workerResults,
+            Calendar startTime,
+            Calendar endTime
```

Comment:
move startTime and endTime to Task. Might need to change it to an abstract class for that.